### ISSUE TITLE
In French, echo and repeat are translated the same way

### ISSUE THREAD

**Issue by clementcontet** [hedyorg/hedy#4715]
**Describe the issue**
In French, `echo` (level 1) and `repeat` (level 6) are translated with the same `répète`.
Even though the levels are not that close and `echo` is not used after level 1 (afaik), I think it can be confusing for students.

**Expected text**
Unfortunately, `répète` is the best translation for both `echo` and `repeat` :(
But:
* `echo` quickly disappears so its good translation is less important
* `repeat` is very close to `répète`
=> So maybe find somehting else for `echo`? What about `rapelle`?

**Comment by etienneCharignon** [comment]
Thank you very much 🙏, `repeat` is at level 7, and I did not reach that level yet. I was about to work on it this week-end.

I agree that `répète` has to be reserved to translate `repeat`. It is too bad for `echo`, where `répète` was working so well.

Propositions : 
- `rappelle`
- `re-dit` (are we allowed to use dash in keywords ?)
- `écho`
- …

**Comment by clementcontet** [comment]
> Propositions :
> 
> - `rappelle`
> - `re-dit` (are we allowed to use dash in keywords ?)
> - `écho`
> - …

Difficult to choose as we know the translation will be worse than `répète` :(
That being said, I find `re-dit` (`say again`) not that good because the `again` implies that the sentence had already been "said".
`écho` is a bit odd because it is not a verb as `echo` is in English.
My preference so far  is `rappelle`!

**Comment by etienneCharignon** [comment]
- `répond`?

That would to : 
```
demande Quel est ton nom 
répond ton nom est
```

**Comment by clementcontet** [comment]
Perfect!

But a French spelling detail:

- est-ce qu'on donne un ordre à Hedy ? `réponds`
- ou bien est-ce qu'on décrit ce que fait Hedy ? "il `répond`"

**Comment by etienneCharignon** [comment]
I chose `réponds`. What do you think?

**Comment by Felienne** [comment]
Will you make this change in Weblate @etienneCharignon? If so, could you mention this issue on the weblate PR in which it is changed so we know it is addressed? Thanks!!

**Comment by etienneCharignon** [comment]
Yes. That's what I was thinking about to do. Unless @clementcontet want to do it before, I will probably do the change in Weblate this week-end.

**Comment by AnneliesVlaar** [comment]
with #6491 this is (hopefully) not an problem anymore
